test/acceptance/cli-test.js

```diff
@@ -1626,7 +1724,8 @@ describe('ember-template-lint executable', function () {
           "app/templates/application.hbs
             1:5  todo  Non-translated string used  no-bare-strings
 
-          ✖ 1 problems (0 errors, 0 warnings, 1 todos)"
+          ✖ 1 problems (0 errors, 0 warnings, 1 todos)
+          ✔ 1 todos created "
```

Comment:
:sob: what do we do about this pluralization? Just ignore it?

I think we've talked about it before...

Reply:
I think so. I considered it, but wanted to keep it consistent with the other output's lack of pluralization.

Reply:
I'd prefer it was consistent than 'right'. If we want to fix all the pluralization, I'd be game (in a separate PR), but consistency > correctness (though one could argue that consistency _is_ correct!).